### ISSUE TITLE
Is there an example to use adversarial training on coqa?

### ISSUE THREAD

**Issue by JayMarx** [stevezheng23/xlnet_extension_tf#83]
hi, thanks for your hard work! Do you known how to add an adversarial training when finetuneing on the coqa dataset?

**Comment by stevezheng23** [comment]
This repo doesn't support adversarial training yet. Normally, for AT, perturbations will be added to input embedding, and based on whether it's AT or VAT, different perturbation updating strategy will be applied when calculate gradients.
